msteams-platform/concepts/resource-specific-consent.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Resource specific consent in Teams
+description: Describes resource specific consent in Teams and how to make advantage of it.
+keywords: teams authorization OAuth SSO AAD rsc
+---
+# Resource specific consent in Teams
+Resource-specific consent (RSC) in Microsoft Teams allows team owners to grant consent to an app to access their team's data and replaces the need for a global admin to approve the app with [tenant-wide admin permissions](/azure/active-directory/manage-apps/grant-admin-consent).
+These permissions include the ability to create, rename and delete channels, read channel messages, create tabs, and add or remove members from a team. This article provides a step-by-step guide to configure RSC in your Teams app.
+> [!Note]
+> To provide consent to your app, the team owner must firstly install the Teams app. Webpages running outside Teams can make RSC Graph calls only after the corresponding Teams app has been installed.
+
+![Consent screen.](/assets/images/rsc/rsc-consentscreen.md)
+
+## Admin Settings
+
+As an admin, you can control whether team owners in your organization can grant team-specific consent (enabled by default) through [settings](https://review.docs.microsoft.com/en-us/MicrosoftTeams/resource-specific-consent?branch=v-lanac-rsc) that you configure in the Azure Active Directory (Azure AD) PowerShell module or the Azure portal and the Microsoft Teams admin center.
+
+
+> **Important**: Changing any of these settings doesn't affect data access for apps that were already granted consent. 
+
+## Configure Teams App to support RSC (Developer)
+
+The basic steps required to configure a service and get a token from the Microsoft identity platform endpoint that your service can use to call Microsoft Graph under its own identity are:
+
+1. Register your application in Azure AD.
+2. Check permissions configured in the 'API Permissions' section.
+3. Update your Teams app manifest to link to your AAD app ID.
+4. Get an access token.
+5. Make a Graph call.
+
+
+#### 1. Register a new application in Azure AD
+
+[Register](/graph/auth-register-app-v2) a new application in the [Azure Active Directory- App Registration portal](https://ms.portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade) to get an AAD app ID. If your AAD app supports [single sign-on (SSO) for tabs](/microsoftteams/platform/tabs/how-to/authentication/auth-aad-sso), then use the same AAD app ID for configuring RSC.
+>[!Note]
+>It is not a good idea to have multiple Teams apps link to the same AAD app ID. Team AAD app ID must be unique – if anyone tries to install and link multiple Teams app to the same AAD app ID, the installation will fail.
+
+#### 2. Check permissions configured in the 'API Permissions' section
```

Comment:
I think this would flow much better if we replaced sections 2 and three with "adding RSC permissions" and "adding non-RSC permissions". The latter being really short.

Reply:
Done